charts/lagoon-logging/templates/logs-dispatcher.servicemonitor.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.logsDispatcher.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "lagoon-logging.logsDispatcher.fullname" . }}
+  labels:
+    {{- include "lagoon-logging.logsDispatcher.labels" . | nindent 4 }}
+    monitoring.lagoon.sh/monitorMe: 'true'
```

Comment:
Interesting, is this currently used?

Reply:
@smlx you mean the `monitoring.lagoon.sh/monitorMe` ? Yes it's used by lagoon-monitoring prometheus
https://github.com/uselagoon/lagoon-charts/pull/133/files#diff-31c0db3ddbe675f8065e5156d8db158ca81b36a321026a4d0488df167841688aR259